apis/Google.Cloud.Dialogflow.V2/Google.Cloud.Dialogflow.V2/Session.cs

```diff
@@ -2059,8 +2063,8 @@ public sealed partial class StreamingDetectIntentRequest : pb::IMessage<Streamin
     public const int SingleUtteranceFieldNumber = 4;
     private bool singleUtterance_;
     /// <summary>
-    /// DEPRECATED. Please use [InputAudioConfig.single_utterance][google.cloud.dialogflow.v2.InputAudioConfig.single_utterance] instead.
-    /// Optional. If `false` (default), recognition does not cease until
+    /// Optional. Please use [InputAudioConfig.single_utterance][google.cloud.dialogflow.v2.InputAudioConfig.single_utterance] instead.
```

Comment:
Going from DEPRECATED to Optional seems wrong, specially because the comment indicating to use something else is still there, which probably means this property has not been unDEPRECATED.

Reply:
It's definitely odd, and I'll follow up in a CL, but I don't think it's a blocker in terms of merging.